packageRpm.nix

```diff
@@ -12,26 +12,56 @@ let
   bin = pkgDesc.bin;
   pkgName = "${project}-${version}-${revision}.${arch}";
   licenseFile = pkgDesc.licenseFile;
+  src = ./.;
 
   writeSpecFile = writeTextFile {
     name = "${project}.spec";
     text = ''
+      %define debug_package %{nil}
+      %define _unpackaged_files_terminate_build 0
       Name:    ${project}
       Version: ${version}
       Release: ${revision}
       Summary: ${pkgDesc.description}
       License: ${pkgDesc.license}
+      BuildArch: ${arch}
+      Source0: ${project}-${version}.tar.gz
```

Comment:
Documentation says that it should be
> Path or URL to the compressed archive of the upstream source code (unpatched, patches are handled elsewhere). This should point to an accessible and reliable storage of the archive, for example, the upstream page and not the packager’s local storage. If needed, more SourceX directives can be added, incrementing the number each time, for example: Source1, Source2, Source3, and so on.

AFAIU, you put the license file and static binary here. Is it ok? The documentation makes me think it should contain source code from `tezos/tezos` :man_shrugging:

Reply:
AFAIR, at this point, `Source` is what package made from. In our case, it is only a static binary, but I agree that it is not entirely correct to package only it without source code. However, it seems that we are not breaking any terms doing it

Reply:
I think it should be ok if we put the link to tezos/tezos as a `Source1`